### PR TITLE
Add support for sensitivity analysis computation

### PR DESCRIPTION
**Pull request by FranckLecuyer**

### CODE REVIEW

**Review by niconoir — COMMENTED** [review]
Minor remarks

File: src/main/java/org/gridsuite/study/server/SensitivityAnalysisService.java [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+public class SensitivityAnalysisService {
+    private String sensitivityAnalysisServerBaseUri;
+
+    private final RestTemplate restTemplate = new RestTemplate();
+
+    private final ObjectMapper objectMapper;
+
+    @Autowired
+    SensitivityAnalysisService(@Value("${backing-services.sensitivity-analysis-server.base-uri:http://sensitivity-analysis-server/}") String sensitivityAnalysisServerBaseUri,
+                               NetworkService networkStoreService,
```
[the remark this answers is not in the document]
Comment (FranckLecuyer) [reply src/main/java/org/gridsuite/study/server/SensitivityAnalysisService.java]: Removed

File: src/main/java/org/gridsuite/study/server/StudyController.java [hunk 2]
```diff
@@ -1242,4 +1242,56 @@ public ResponseEntity<Void> notify(@PathVariable("studyUuid") UUID studyUuid,
...
+        return result != null ? ResponseEntity.ok().body(result) :
+            ResponseEntity.noContent().build();
+    }
+
+    @GetMapping(value = "/studies/{studyUuid}/nodes/{nodeUuid}/sensitivity-analysis/status")
+    @Operation(summary = "Get the sensitivity analysis status on study")
+    @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "The sensitivity analysis status"),
+        @ApiResponse(responseCode = "204", description = "No sensitivity analysis has been done yet"),
+        @ApiResponse(responseCode = "404", description = "The sensitivity analysis status has not been found")})
+    public ResponseEntity<String> getsensitivityAnalysisStatus(@Parameter(description = "Study UUID") @PathVariable("studyUuid") UUID studyUuid,
```
[the remark this answers is not in the document]
Comment (FranckLecuyer) [reply src/main/java/org/gridsuite/study/server/StudyController.java]: Changed

File: src/main/java/org/gridsuite/study/server/StudyController.java [hunk 3]
```diff
@@ -1242,4 +1242,56 @@ public ResponseEntity<Void> notify(@PathVariable("studyUuid") UUID studyUuid,
...
+                                                               @Parameter(description = "nodeUuid") @PathVariable("nodeUuid") UUID nodeUuid) {
+        String result = studyService.getSensitivityAnalysisStatus(nodeUuid);
+        return result != null ? ResponseEntity.ok().body(result) :
+            ResponseEntity.noContent().build();
+    }
+
+    @PutMapping(value = "/studies/{studyUuid}/nodes/{nodeUuid}/sensitivity-analysis/stop")
+    @Operation(summary = "stop sensitivity analysis on study")
+    @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "The sensitivity analysis has been stopped")})
+    public ResponseEntity<Void> stopsensitivityAnalysis(@Parameter(description = "Study uuid") @PathVariable("studyUuid") UUID studyUuid,
```
[the remark this answers is not in the document]
Comment (FranckLecuyer) [reply src/main/java/org/gridsuite/study/server/StudyController.java]: Changed

**Review by niconoir — APPROVED** [review]

**Review by geofjamg — CHANGES_REQUESTED** [review]
Same as for sensitivity analysis server, quad -> branch

**Review by geofjamg — APPROVED** [review]

**Pull request merged**